src/applications/personalization/profile360-beta/components/ProfileView.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
```

Comment:
aside from ungating a couple of components, this file is identical to the existing [ProfileView](https://github.com/department-of-veterans-affairs/vets-website/blob/57b5cfb63c7e9abd775432d2c57ebcc46aef076e/src/applications/personalization/profile360/components/ProfileView.jsx#L1)